src/js/components/ComboBox/Combobox.tsx

```diff
@@ -322,6 +325,7 @@ const ComboBox = ({
                   }}
                   mods="u-spaceRightMd u-colorNeutral7"
                   type="link"
+                  testId="combobox-cancel-button"
```

Comment:
We should have probably added a name to the test id because often we use multiple combo boxes

It could have been something like `combobox-cancel-button-season` or `combobox-season-cancel-button` so we don't have to deal with potentially finding multiple instances of `combobox-cancel-button` when we use multiple options